src/main/java/com/capitalone/dashboard/model/Repo.java

```diff
@@ -0,0 +1,38 @@
+package com.capitalone.dashboard.model;
+
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "helm_repo")
```

Comment:
we need to come up with data model that is tool agnostic

Reply:
Resolved